src/rdb.c

```diff
@@ -33,11 +33,24 @@ void Restore_Globals() {
 }
 
 void Discard_Globals_Backup() {
-  Indexes_Free(specDict_g_bkup);
+  // save global to temp
+  dict *specDict_g_temp = specDict_g;
```

Comment:
Can you add a comment that explains why we are doing it and that this is a temporary solution until we will have a more robust one?